### DISCUSSION TITLE
Show page modifications in Slack preview

### DISCUSSION THREAD

**Discussion by daladim** [outline/outline#4695] [Ideas]
Thanks a lot for version 0.67!
The highlighted diff in the history is a great enhancement.

I'm connecting Outline to a Slack channel so that it posts notifications on page edition. These messages contain a preview of the page, that always shows the first line.
This preview is not terribly helpful, and it would be great that it rather contained the line(s) that have been edited instead.

**Comment by daladim** [comment]
I'm not sure how easily that could fit into the Slack model however. The preview shown in Slack is limited to a few lines only, and I'm not sure how you could distinguish addition from removal in such a limited amount of space.

**Reply by tommoor** [reply]
Agreed, created an associated issue
